apps/condo/domains/organization/schema/FindOrganizationsByTinLog.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Generated by `createschema organization.FindOrganizationsByTinLog 'user:Relationship:User:SET_NULL; tin:Text'`
+ */
+
+const get = require('lodash/get')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { versioned, uuided, tracked, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/organization/access/FindOrganizationsByTinLog')
+
+
+const ERRORS = {
+    USER_IS_MISSING: {
```

Comment:
`user` is required field, won't there be an error (default) if you don't pass the required field?

Reply:
we need this check